modules/notes.js

```diff
@@ -4,6 +4,7 @@ const { lambda, getFuncName } = require('./util')
 
 
 const DEV_CHANNEL_ID = 'G1FDULP1R'
+const DATA_CHANNEL_ID = 'G8N74MDCH'
```

Comment:
not a problem about removing this and changing the check Maliha. I guess the hard coded approach was more like a lazy way to try it out and see if we would end up using this automation